app/controllers/api/v1/parature_faq_controller.rb

```diff
@@ -1,4 +1,4 @@
 class Api::V1::ParatureFaqController < ApplicationController
   include Searchable
-  search_by :question, :answer, :update_date_start, :update_date_end, :countries, :industries, :q, :topics
+  search_by :question, :answer, :update_date, :countries, :industries, :q, :topics
```

Comment:
@tmhammer removing the "update_date_start" and "update_date_end" params from this v1 endpoint will break the interface. I don't think we can do that. @ErikSArnold ?